### ISSUE TITLE
Controllers replace qualifier with import

### ISSUE THREAD

**Issue by typedin** [typedin/laravel-calendly#9]
Considering this file:
[src/Http/Controllers/CalendlyEventTypesController.php](https://github.com/typedin/laravel-calendly/blob/7c86f44d930ad8c8dc4edc3bc837e832b9a14922/src/Http/Controllers/CalendlyEventTypesController.php)

Those pieces of code should have their qualifiers replaced with import
```PHP
$all = collect($response['collection'])->mapInto(EventType::class)->all();
```

```PHP
return response()->json([
    'event_type' => new \Typedin\LaravelCalendly\Entities\EventType($response),
]);
```

**Comment by typedin** [comment]
Rector should do the job. I just need to figure out how to securely call rector through the CI.

**Comment by typedin** [comment]
Part of CI, closed by cf9d9c894498ce9b0e22d7386b0a6ac014f917cc
